app/models/project.rb

```diff
@@ -253,11 +253,6 @@ def project_directory_pathname
       end
     end
 
-    # Ensure that the project directory is a valid path
-    def safe_directory(directory)
-      Project.safe_directory(directory)
-    end
-
```

Comment:
This method was unused (and thank goodness for that because it would have been an infinite recursive call!)